bitrepository-audit-trail-service/src/test/java/org/bitrepository/audittrails/preserver/StubAuditEventIterator.java

```diff
@@ -0,0 +1,37 @@
+package org.bitrepository.audittrails.preserver;
+
+import org.bitrepository.audittrails.store.AuditEventIterator;
+import org.bitrepository.bitrepositoryelements.AuditTrailEvent;
+import org.bitrepository.bitrepositoryelements.FileAction;
+import org.bitrepository.common.utils.CalendarUtils;
+
+import java.math.BigInteger;
+
+/**
+ * AuditEventIterator that will only return an event on first iteration.
+ */
+public class StubAuditEventIterator extends AuditEventIterator {
+    boolean called = false;
+    public StubAuditEventIterator() {
+        super(null);
+    }
+
+    @Override
+    public AuditTrailEvent getNextAuditTrailEvent() {
+        String PILLAR_ID = "pillarID";
+        String ACTOR = "actor";
```

Comment:
Where I learned, all uppercase was reserved for `static final`  “variables”. Would we prefer `pillarId` and `actor`?

Reply:
Yeah, this was some old stuff I lazily moved around without renaming, my bad. Good catch :+1: